docs/source/en/api/pipelines/consistency_models.mdx

```diff
@@ -1,87 +1,49 @@
-- *ct_cat256 (256x256 resolution)* [openai/diffusers-ct_cat256](https://huggingface.co/openai/diffusers-ct_cat256)
+## Tips
 
-## Available Pipelines
+For an additional speed-up, use `torch.compile` to generate multiple images in <1 second:
 
-| Pipeline | Tasks | Demo | Colab |
-|:---:|:---:|:---:|:---:|
-| [ConsistencyModelPipeline](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/pipeline_consistency_models.py) | *Unconditional Image Generation* | | |
+```diff
+  import torch
+  from diffusers import ConsistencyModelPipeline
 
-This pipeline was contributed by our community members [dg845](https://github.com/dg845) and [ayushtues](https://huggingface.co/ayushtues) ❤️
+  device = "cuda"
+  # Load the cd_bedroom256_lpips checkpoint.
+  model_id_or_path = "openai/diffusers-cd_bedroom256_lpips"
+  pipe = ConsistencyModelPipeline.from_pretrained(model_id_or_path, torch_dtype=torch.float16)
+  pipe.to(device)
 
-## Usage Example
++ pipe.unet = torch.compile(pipe.unet, mode="reduce-overhead", fullgraph=True)
 
-```python
-import torch
-
-from diffusers import ConsistencyModelPipeline
-
-device = "cuda"
-# Load the cd_imagenet64_l2 checkpoint.
-model_id_or_path = "openai/diffusers-cd_imagenet64_l2"
-pipe = ConsistencyModelPipeline.from_pretrained(model_id_or_path, torch_dtype=torch.float16)
-pipe.to(device)
-
-# Onestep Sampling
-image = pipe(num_inference_steps=1).images[0]
-image.save("consistency_model_onestep_sample.png")
-
-# Onestep sampling, class-conditional image generation
-# ImageNet-64 class label 145 corresponds to king penguins
-image = pipe(num_inference_steps=1, class_labels=145).images[0]
-image.save("consistency_model_onestep_sample_penguin.png")
-
-# Multistep sampling, class-conditional image generation
-# Timesteps can be explicitly specified; the particular timesteps below are from the original Github repo.
-# https://github.com/openai/consistency_models/blob/main/scripts/launch.sh#L77
-image = pipe(timesteps=[22, 0], class_labels=145).images[0]
-image.save("consistency_model_multistep_sample_penguin.png")
+  # Multistep sampling
+  # Timesteps can be explicitly specified; the particular timesteps below are from the original Github repo:
+  # https://github.com/openai/consistency_models/blob/main/scripts/launch.sh#L83
+  for _ in range(10):
+      image = pipe(timesteps=[17, 0]).images[0]
+      image.show()
 ```
 
-For an additional speed-up, one can also make use of `torch.compile`. Multiple images can be generated in <1 second as follows:
+<Tip>
 
-```py
-import torch
-from diffusers import ConsistencyModelPipeline
+Make sure to check out the Schedulers [guide](/using-diffusers/schedulers) to learn how to explore the tradeoff between scheduler speed and quality, and see the [reuse components across pipelines](/using-diffusers/loading#reuse-components-across-pipelines) section to learn how to efficiently load the same components into multiple pipelines.
```

Comment:
Consistency models may not have this guarantee always, no, @patrickvonplaten?

Reply:
Agree! Let's correct this. Consistency models should always only use the same consistency scheduler for now